### ISSUE TITLE
The issue has returned 

### ISSUE THREAD

**Issue by warmfart** [alievk/avatarify-desktop#50]
Issue is back

**Comment by warmfart** [comment]
![Uploading image.jpg…]()

**Comment by DG12X** [comment]
please can you leave your email, telegram or any platform to get the video for the *solving environment* fix. please i need it thanks.

**Comment by DG12X** [comment]
> please can you leave your email, telegram or any platform to get the video for the solving environment fix. please i need it thanks.

**Comment by JohanAR** [comment]
Uploading the video to youtube and posting the link here seems like the easiest way to do it.

**Comment by DG12X** [comment]
> Uploading the video to youtube and posting the link here seems like the easiest way to do it.

yes that will do too or better still upload it on mega cloud and paste the link to download the video. anything please cause i am stuck right now.

**Comment by Echolink50** [comment]
From dealing with the python version, I think the issue with RTX 30 series cards is the pytorch and cudatoolkit version. 11+ is for newer cards. Seems an easy enough fix for somebody that knows how to compile this stuff
